index.src.html

```diff
@@ -467,9 +470,8 @@ <h3 dfn export id="referrer-policy-empty-string">The empty string</h3>
 
   The empty string "" corresponds to no <a for="/">referrer policy</a>, causing a
   fallback to a <a for="/">referrer policy</a> defined elsewhere, or in the case where
-  no such higher-level policy is available, defaulting to
-  <a>"<code>no-referrer-when-downgrade</code>"</a>. This defaulting happens in
-  the [[#determine-requests-referrer]] algorithm.
+  no such higher-level policy is available, falling back to the <a>default referrer
+  policy</a>. This happens in Fetch's main fetch algorithm, for example.
```

Comment:
Where else would it happen?

Reply:
@mikewest I'm working on shepherding this change through the specs. Could you help answer @annevk 's question? I'm not sure if this is purely a clarification question or if it needs to result in a change in this PR?